specs/DefaultDownloadDialog.md

```diff
@@ -0,0 +1,272 @@
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_RIGHT,
+
+} COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION;
+
+/// The padding is the distance between the WebView corner specified by the
+/// `DefaultDownloadDialogPosition` and the default download dialog corner
+/// nearest to it. It is measured in raw pixels or logical pixels, depending on
+/// the COREWEBVIEW2_BOUNDS_MODE. Use (0, 0) to align the dialog to the WebView
+/// corner with no padding.
+typedef struct COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_PADDING {
+
+  /// The padding in the x-direction from the `DefaultDownloadDialogPosition`.
+  UINT32 x;
+
+  /// The padding in the y-direction from the `DefaultDownloadDialogPosition`.
+  UINT32 y;
+
+} COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_PADDING;
+
+[uuid(9139c04d-8f37-42ae-8b63-01940c34d22f), object, pointer_default(unique)]
+interface ICoreWebView2_6 : ICoreWebView2_5
+{
+  // Raised when the `IsDefaultDownloadDialogOpen` property changes.
+  HRESULT add_IsDefaultDownloadDialogOpenChanged(
+      [in] ICoreWebView2IsDefaultDownloadDialogOpenChangedEventHandler*
+          handler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with
+  /// `add_IsDefaultDownloadDialogOpenChanged`.
+  HRESULT remove_IsDefaultDownloadDialogOpenChanged(
+      [in] EventRegistrationToken token);
+
+  /// `TRUE` if the default download dialog is currently open. Toggling WebView
+  /// visibility also toggles default download dialog visibility.
+  [propget] HRESULT IsDefaultDownloadDialogOpen(
+      [out, retval] BOOL* isDefaultDownloadDialogOpen);
+
+  /// Open the default download dialog. If the dialog is opened before there
+  /// are recent downloads, the dialog shows all past downloads for the
+  /// current profile. Otherwise, the dialog shows only the recent downloads
+  /// with a "See more" button for past downloads.
+  ///
+  /// \snippet ViewComponent.cpp ToggleDefaultDownloadDialog
+  HRESULT OpenDefaultDownloadDialog();
+
+  /// Close the default download dialog.
+  HRESULT CloseDefaultDownloadDialog();
+
+  /// Get the default download dialog position.
+  [propget] HRESULT DefaultDownloadDialogPosition(
+      [out, retval] COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION* position);
+
+  /// Set the default download dialog position relative to the WebView bounds.
+  /// The dialog can be positioned against any of the WebView corners (see
+  /// COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION). When the WebView or dialog
+  /// changes size, the dialog keeps its position relative to the corner. The
+  /// dialog may become partially or completely outside of the WebView bounds if
+  /// the WebView is small enough. Set the padding from the position with the
+  ///  `DefaultDownloadDialogPadding` property.
+  ///
+  /// \snippet ViewComponent.cpp SetDefaultDownloadDialogPosition
+  [propput] HRESULT DefaultDownloadDialogPosition(
+      [in] COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION position);
```

Comment:
Parameter names for property methods should be `value`. Also for the other properties here.
```suggestion
      [in] COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION value);
```